cloudbuild.yaml

```diff
@@ -143,23 +177,105 @@ steps:
         -var=gcs_bucket=gke-aieco-jupyter-$SHORT_SHA \
         -auto-approve -no-color
 
-        cd /workspace/applications/ray/
+    allowFailure: true
+    waitFor: ['test jupyterhub']
+
+  - id: 'test rag'
+    name: 'gcr.io/$PROJECT_ID/terraform'
+    entrypoint: 'sh'
+    args:
+      - '-c'
+      - |
+        set -e
+
+        # Get kube config
+        gcloud container clusters get-credentials \
+        ml-$SHORT_SHA-$_PR_NUMBER-cluster \
+        --location $_REGION \
+        --project $PROJECT_ID 
+
+        cd /workspace/modules/jupyter/tests
+        python3 change_jupyter_config.py
+
+        cd /workspace/applications/rag/
+        terraform init
```

Comment:
I think you can remove this, `terraform init` was run in the `validate rag` step

Reply:
Removed.